cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -0,0 +1,298 @@
+"""
+This tool allows you to partially clone a data from courtlistener.com to your
+local environment, you only need to pass the type and object id and run it.
+
+e.g.
+
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Opinion --id 9355884
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Docket --id 5377675
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Person --id 16207
+docker-compose -f docker/courtlistener/docker-compose.yml exec cl-django python manage.py clone_from_cl --type Court --id usnmcmilrev
+
+
+This tool is only for development purposes, to enable it you need to set
+environment variable DEVELOPMENT to True. Also need to set CL_API_TOKEN env
+variable.
+
+This is still work in progress, some data is not cloned yet.
+
+"""
+
+import os
+import sys
+
+import environ
+import requests
+from django.db import transaction
+from django.utils.dateparse import parse_date
+
+from cl.lib.command_utils import VerboseCommand
+from cl.people_db.models import Person
+from cl.search.models import Citation, Court, Docket, Opinion, OpinionCluster
+from cl.search.tasks import add_items_to_solr
+
+env = environ.FileAwareEnv()
+DEVELOPMENT = env.bool("DEVELOPMENT", default=False)
+
+VALID_TYPES = ("Opinion", "Docket", "Person", "Court")
```

Comment:
I think this was done in cl_update_index this way because solr required the correct model to index the item and query the id to correct object class, in this specific situation i think we don't need to that exactly the same because we are working with django, so we can do the imports directly instead of using get_model()

Reply:
Yeah, the solr one uses this in a bit of a clever way. Here it's nice to do just to be consistent across scripts so that when any script needs a model as an input, you can be sure it needs something like `search.Xyz`